makefile

```diff
@@ -1,8 +1,11 @@
-.PHONY: test docs develop build publish publish_docs
+.PHONY: test docs develop build publish publish_docs lint
 
 develop:
 	pipx run --spec=git+https://github.com/cs01/nox.git@5ea70723e9e6 nox -s develop
 
+lint:
+	pipx run --spec=git+https://github.com/cs01/nox.git@5ea70723e9e6 nox -s lint
```

Comment:
the fork of nox is only needed when creating venvs in the nox session. lint doesn't need it, it can just do `pipx run nox -s lint`.